src/extractors/custom/wired.jp/index.js

```diff
@@ -0,0 +1,41 @@
+import URL from 'url';
+
+export const WiredJpExtractor = {
+  domain: 'wired.jp',
+
+  title: {
+    selectors: ['h1.post-title'],
+  },
+
+  author: {
+    selectors: ['p[itemprop="author"]'],
+  },
+
+  date_published: {
+    selectors: ['time'],
```

Comment:
Let's take advantage of the fact that this is a `time` tag and use the `datetime` property instead, that way we wouldn't have to define a custom format

Reply:
Thank you.
I fixed.